mdata/store_cassandra.go

```diff
@@ -391,19 +402,26 @@ func (c *CassandraStore) processReadQueue() {
 
 // Basic search of cassandra in the table for given ttl
 // start inclusive, end exclusive
-func (c *CassandraStore) Search(key string, ttl, start, end uint32) ([]chunk.IterGen, error) {
+func (c *CassandraStore) Search(ctx context.Context, key string, ttl, start, end uint32) ([]chunk.IterGen, error) {
 	table, err := c.getTable(ttl)
 	if err != nil {
 		return nil, err
 	}
-	return c.SearchTable(key, table, start, end)
+	return c.SearchTable(ctx, key, table, start, end)
 }
 
 // Basic search of cassandra in given table
 // start inclusive, end exclusive
-func (c *CassandraStore) SearchTable(key, table string, start, end uint32) ([]chunk.IterGen, error) {
+func (c *CassandraStore) SearchTable(ctx context.Context, key, table string, start, end uint32) ([]chunk.IterGen, error) {
+	ctx, span := tracing.NewSpan(ctx, c.tracer, "CassandraStore.SearchTable")
```

Comment:
same here, unused `ctx`

Reply:
actually that probably really doesn't matter much since that memory of `ctx` is already allocated anyway. the only difference is that by using a `_` instead of assigning to `ctx` we could save a few ops for overwriting that memory, which is probably not significant